lumen/ai/prompts/sql_query.jinja2

```diff
@@ -9,16 +9,19 @@ The data for table '{{ table }}' follows the following JSON schema:
 {% if not join_required %}
 It was already determined that no join is required, so only use the existing
 table '{{ table }}' to calculate the result.
+{% else %}
+If the join's values do not align 50% of the time, then perform a join based on the
```

Comment:
How would it know whether it's likely to align?

Reply:
Potentially schema enums (I'll have to double check dates) or min/max

Reply:
Should be `min/max`.

Reply:
Seems to work with min/max
<img width="992" alt="image" src="https://github.com/user-attachments/assets/b1b5f217-1a28-4085-95ed-fcc524e73482">

Reply:
Seems to work with string enums too 

<img width="1149" alt="image" src="https://github.com/user-attachments/assets/61ce0c76-1b01-4c5b-8491-d30de14aa014">

Reply:
And lastly for numeric matching min/max

<img width="849" alt="image" src="https://github.com/user-attachments/assets/4c8620d1-ccb4-4e7a-8507-68fc4eec9ba4">